graspy/match/faq.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import numpy as np
+import math
+import matplotlib.pyplot as plt
+from scipy.optimize import linear_sum_assignment
+from scipy.optimize import minimize_scalar
+from scipy.optimize import minimize
+from sinkhorn_knopp import sinkhorn_knopp as skp
+
+sk = skp.SinkhornKnopp()
+
+
+class FastApproximateQAP:
+    """
+    Fast Approximate QAP Algorithm (FAQ)
```

Comment:
everything in the docstring should be less than 88 characters, normally black would fix this but it won't for docstrings

Reply:
still need to do this